src/app/screens/App/shared/components/LoggedOutHeader.js

```diff
@@ -1,37 +1,34 @@
 import React from 'react';
-import { Navbar, Nav, NavItem, Button, FormGroup, FormControl, InputGroup } from 'react-bootstrap';
+import { Navbar, Nav, NavItem, Button, FormGroup, FormControl } from 'react-bootstrap';
 import { LinkContainer } from 'react-router-bootstrap';
 
 const LoggedOutHeader = () => (
     <Navbar>
         <Nav>
-            <NavItem disabled>Personal</NavItem>
-            <NavItem disabled>Open source</NavItem>
-            <NavItem disabled>Business</NavItem>
-            <NavItem disabled>Explore</NavItem>
-            <NavItem disabled>Help</NavItem>
+            <NavItem>Personal</NavItem>
+            <NavItem>Open Source</NavItem>
+            <NavItem>Business</NavItem>
+            <NavItem>Explore</NavItem>
+            <NavItem>Personal</NavItem>
         </Nav>
+        <Navbar.Form pullRight>
+            <FormGroup>
```

Comment:
This isn't actually in a form. It's just a form group which is bootstraps way of styling elements that are traditionally in forms. If you actually wanted it to be in a form with actions you would have to actually wrap it in a form tag.